test/unit/uk/gov/hmrc/bindingtariffclassification/model/CaseSearchTest.scala

```diff
@@ -37,7 +36,7 @@ class CaseSearchTest extends UnitSpec {
     queueId = Some("queue-id"),
     eori = Some("eori-number"),
     assigneeId = Some("assignee-id"),
-    statuses = Some(Set(NEW, OPEN)),
+    statuses = Some(Set(PseudoCaseStatus.NEW, PseudoCaseStatus.OPEN)),
```

Comment:
Did you deliberately not import PseudoCaseStatus to highlight in the code that these are not regular statuses?

Reply:
Could easily import PseudoCaseStatus I guess